addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -824,10 +824,12 @@ registry.backgroundVideo = publicWidget.Widget.extend({
         // YouTube does not allow to auto-play video in mobile devices, so we
         // have to play the video manually.
         if (this.isMobileEnv && this.isYoutubeVideo) {
-            new window.YT.Player(this.iframeID, {
-                events: {
-                    onReady: ev => ev.target.playVideo(),
-                }
+            window.YT.ready(() => {
```

Comment:
Should this just be waiting for `onYouTubeIframeAPIReady`, which is documented and already used in this file?
https://developers.google.com/youtube/iframe_api_reference#Requirements

I could not reproduce the issue so let me know

Reply:
Thanks for your response.

I couldn't reproduce it either. Even on the customer database, it doesn't occur every time. I guess in exceptional circumstances, it should load it before the other part. 

Could you please let me know what your suggestion is?

Reply:
We cannot just introduce some code hoping it will solve the issue. Especially this kind of undocumented code found on [a 10 upvote post on stackoverflow](https://stackoverflow.com/questions/52062169/uncaught-typeerror-yt-player-is-not-a-constructor).
This is 13.0, possibly hundred of thousand DBs using it.
We can't ship code hoping for the best, that might just break everything.

Generally speaking, if we can't reproduce the issue on runbot (or local, or whatever), this does not qualifies as a bug that should be fixed.

In this case:
1. We should identify when, where and why it breaks, so we are able to reproduce it.
2. Provide a fix that solves the issue, and ensure everything else is still working (in this case, desktop etc)

Did you even checked if it fixed the client database? If yes, that's a start, and we can know check if the "documented and already used" `onYouTubeIframeAPIReady` I suggest also fix it -> If yes, it would be better.

Reply:
Thanks for your explanation.

Yes, this fixed the problem with the customer database. But I realized that there was another problem with their theme, which after fixing it, the main problem was solved.

Reply:
I am sorry but I am not sure what you exactly mean or if you are asking something here?

> Yes, this fixed the problem with the customer database.

Which fix? The YT.ready or the onYouTubeIframeAPIReady?

> But I realized that there was another problem with their theme

What problem :shrug: ?

> which after fixing it

The new bug related to their theme?

> the main problem was solved

The YT mobile error? So we don't need any fix at all and it was a bug induced by the new theme issue you found and fixed?

Thanks..

Reply:
YT.ready was solved the client problem, but after upgrading the base module, the problem was fixed on the client database, without YT.ready.

Also, I couldn't reproduce the error, so I think we don't need any fix. 

Regards,

Reply:
Ok :shrug: 
In any case, if you were to work on this again later, don't forget to check the `onYouTubeIframeAPIReady`